rcldotnet/QosProfile.cs

```diff
@@ -0,0 +1,29 @@
+
+
+namespace ROS2.Utils
+{
+    public static class QosProfile
+    {
+        /// <summary> QoS presets that encompass the majority of use-cases for ROS2 messages
+        /// for more details on the different policies and profile use cases, see:
+        /// https://index.ros.org/doc/ros2/Concepts/About-Quality-of-Service-Settings/ </summary>
+        public enum Profile 
+        {
+            /// <summary> Prioritizes timely readings, rather than 
+            /// ensuring they all arrive. Best effort reliability and smaller queue depth. </summary>
+            SENSOR_DATA = 0,
```

Comment:
Should these be be formatted as C# enums? (like SensorData or ProfileParameters)